src/helpers/utils.ts

```diff
@@ -412,3 +412,24 @@ export const execute = (cmd: string, cwd?: string): Promise<{stderr: string; std
     cwd,
     maxBuffer: 5 * 1024 * 5000,
   })
+
+type GitHubWebhookPayload = {
+  pull_request: {
+    head: {
+      sha: string
+    }
+  }
+}
+
+export const getGitHeadShaFromGitHubWebhookPayload = () => {
+  if (!process.env.GITHUB_EVENT_PATH) {
+    return ''
+  }
+  try {
+    const parsedContents = JSON.parse(readFileSync(process.env.GITHUB_EVENT_PATH, 'utf8')) as GitHubWebhookPayload
+
+    return parsedContents.pull_request.head.sha
+  } catch (e) {
+    return ''
```

Comment:
Can we send a log message to indicate we could not parse the payload?

Reply:
added in https://github.com/DataDog/datadog-ci/pull/1361/commits/5efc53ad6c5ab8d10e189f290870971f0e9fcbdc